### ISSUE TITLE
MMDeploy Compatibility

### ISSUE THREAD

**Issue by austinmw** [open-mmlab/mmtracking#592]
Hi, I have a NVIDIA Triton server that’s running a TensorRT engine file of YOLOX I generated using MMDeploy. 

I’m wondering, is it possible to add ByteTrack or another multi-object tracker to this pipeline?

**Comment by dyhBUPT** [comment]
Yes, you can deploy trackers from mmtracking into your pipeline.

For ByteTrack and SORT, because you have already deployed the detector, you just need to call the method `track` in `xxxtracker`. For some other trackers, you may need to deploy other components, e.g., reid.

BTW, for ByteTrack, it has a deploy tutorial in its [official code](https://github.com/ifzhang/ByteTrack).

**Comment by austinmw** [comment]
Thanks!
